items.json

```diff
@@ -22008,17 +22008,17 @@
     "name": "Shadowfax",
     "culture": "Neutral",
     "type": "Mount",
-    "price": 19395,
+    "price": 22635,
     "weight": 420.0,
-    "tier": 11.5202217,
+    "tier": 12.5331268,
     "requirement": 0,

```

Comment:
too high

Reply:
Shadowfax only gained +9 HP, sitting at 216 now.

Reply:
i'd like you to still keep tier below tier 12. I already made an exception for you to have above tier 10 mounts , but we can't have tier inflating indefinetely

Reply:
Yeah that's fair enough, but could anything be done to accommodate these changes in terms of:

1) Potentially devaluing HP in the calculator so the health rejig isn't pushing these mounts to super high tiers over moderate HP increases? Because I think these are needed, particularly given the damage cavalry sustains and the playstyles of these different mounts, a bigger difference between relevant archetypes would fit this context and help players make choices about which mounts they should use and for what playstyle. 

2) Perhaps bringing speed down in value, though honestly I'm not sure how the balancing act would work out in detail here. Would have to be a bit of trial and error again. What I would want to see is something as described below -

If you could try and bring these two mounts closer together in tier (though this will probably be difficult) and more importantly, both under tier 12, that would be the ideal:
  {
    "id": "crpg_mount1_maneuverable_14",
    "name": "Baucent",
    "price": 19758,
    "tier": 11.6376371,
      "bodyLength": 95,
      "chargeDamage": 5,
      "maneuver": 86,
      "speed": 46,
      "hitPoints": 198,
  },
Highest maneuver, good speed, low HP and charge. The main factor here is the large amount of both maneuver and speed.

  {
    "id": "crpg_mount_greathorse_15",
    "name": "Lionheart",
    "price": 26031,
    "tier": 13.5200825,
      "bodyLength": 107,
      "chargeDamage": 9,
      "maneuver": 84,
      "speed": 42,
      "hitPoints": 249,
  },
Good maneuver, slow speed, good HP and charge (both disaffected however by the lower speed). Pretty good stats all round on paper, but the main factor is the low speed.

I think this Lionheart, with +2 Speed to the great horse line, is a necessary change and as part of the health re-jig it would also receive a buff of +12 HP, to distinguish its slower moving, closer combat role. Stats shown above.